methods/linkedList/linkedList.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+const algolib = require('../index.js');
+
+class LinkedList {
+
+    constructor(){
+        this._elements = [];
```

Comment:
Your Linked List actually is an Array with linked objects. But it breakes all concept of LL, because in LL elements are not ordered in memory.